insights/tests/parsers/test_grubenv.py

```diff
@@ -31,6 +31,10 @@
 ######################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################
```

Comment:
I believe these kind of lines should be removed after the spec changed to command.